src/backends/compose/compose_backend.py

```diff
@@ -436,26 +439,21 @@ def add_lnd_service(self, tank, services):
             "--bitcoin.node=bitcoind",
             f"--bitcoind.rpcuser={tank.rpc_user}",
             f"--bitcoind.rpcpass={tank.rpc_password}",
-            f"--bitcoind.rpchost={tank.ipv4}:{tank.rpc_port}",
-            f"--bitcoind.zmqpubrawblock=tcp://{tank.ipv4}:{tank.zmqblockport}",
-            f"--bitcoind.zmqpubrawtx=tcp://{tank.ipv4}:{tank.zmqtxport}",
-            f"--externalip={tank.lnnode.ipv4}",
+            f"--bitcoind.rpchost={bitcoind_container_name}:{tank.rpc_port}",
+            f"--bitcoind.zmqpubrawblock=tcp://{bitcoind_container_name}:{tank.zmqblockport}",
+            f"--bitcoind.zmqpubrawtx=tcp://{bitcoind_container_name}:{tank.zmqtxport}",
```

Comment:
in "backend: remove ipv4 from compose backend" (https://github.com/bitcoin-dev-project/warnet/commit/50107b7f1dd88a1f0d393ed0e64b2ef8b19fddc7):

This works because every bitcoind node is behind a service, right?